src/index.tsx

```diff
@@ -10,14 +10,16 @@ interface Item {
 interface ReactAccordionProps {
   items: Item[]
   // eslint-disable-next-line react/require-default-props
-  onClick?: (event: SyntheticEvent, index: number) => void
+  onClick?: (event: SyntheticEvent, index: number) => void,
+  // eslint-disable-next-line react/require-default-props
+  className?: string,
```

Comment:
Is this required? I seem to remember className being a sort of "automatically assumed" prop....

Reply:
I'm not sure about typescript. I'd assume (hope) you have to explicitly set it. I don't like magic. :confused: